components/ILIAS/Certificate/Table/CertificateOverviewTable.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+/**
+ * This file is part of ILIAS, a powerful learning management system
+ * published by ILIAS open source e-Learning e.V.
+ *
+ * ILIAS is licensed with the GPL-3.0,
+ * see https://www.gnu.org/licenses/gpl-3.0.en.html
+ * You should have received a copy of said license along with the
+ * source code, too.
+ *
+ * If this is not the case or you just want to try ILIAS, you'll find
+ * us at:
+ * https://www.ilias.de
+ * https://github.com/ILIAS-eLearning
+ *
+ *********************************************************************/
+
+declare(strict_types=1);
+
+namespace ILIAS\Certificate\Table;
+
+use DateTime;
+use DateTimeImmutable;
+use Exception;
+use Generator;
+use ilAccessHandler;
+use ilCtrl;
+use ilCtrlInterface;
+use ILIAS\Data\Order;
+use ILIAS\Data\Range;
+use ILIAS\UI\Component\Table\DataRetrieval;
+use ILIAS\UI\Component\Table\DataRowBuilder;
+use ILIAS\UI\Factory;
+use ILIAS\UI\Implementation\Component\Input\Container\Filter\Standard;
+use ILIAS\UI\Implementation\Component\Table\Table;
+use ILIAS\UI\Renderer;
+use ILIAS\UI\URLBuilder;
+use ILIAS\UI\URLBuilderToken;
+use ilLanguage;
+use ilLink;
+use ilObjCertificateSettingsGUI;
+use ilObject;
+use ilObjUser;
+use ilUIService;
+use ilUserCertificate;
+use ilUserCertificateRepository;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ServerRequestInterface;
+
+class CertificateOverviewTable implements DataRetrieval
```

Comment:
Can we switch the folder to `AdministrationOverview` or `Overview` and adapt the namespace accordingly?